subsys/bluetooth/host/classic/l2cap_br.c

```diff
@@ -526,6 +545,13 @@ static int l2cap_br_info_rsp(struct bt_l2cap_br *l2cap, uint8_t ident,
 			err = -EINVAL;
 			break;
 		}
+		/*
+		 * 2.1 CHANNEL IDENTIFIERS in
+		 * BLUETOOTH CORE SPECIFICATION Version 5.4 | Vol 3, Part A.
+		 * The range of fixed L2CAP CID is 0x0001 ~ 0x0007 both for LE and BR.
+		 * So use one octet buffer to keep the `Fixed channels supported`
+		 * of peer device.
+		 */
```

Comment:
This is indeed a good place for the clarification. I would also add here a note that the field is actually 8 bytes but the current implementation only cares about the first one. It could also make sense to add a similar note to the respective struct member definition in `struct bt_l2cap_br`.

Reply:
Done.